cmdstanpy/install_cmdstan.py

```diff
@@ -161,23 +135,12 @@ def compile_example() -> None:
             os.path.join('examples', 'bernoulli', 'bernoulli' + EXTENSION)
         ).as_posix(),
     ]
-    proc = subprocess.Popen(
-        cmd,
-        cwd=None,
-        stdin=subprocess.DEVNULL,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        env=os.environ,
-    )
-    while proc.poll() is None:
-        if proc.stdout:
-            proc.stdout.readline().decode('utf-8')
-    _, stderr = proc.communicate()
-    if proc.returncode:
-        msgs = ['Failed to compile example model bernoulli.stan']
-        if stderr:
-            msgs.append(stderr.decode('utf-8').strip())
-        raise CmdStanInstallError('\n'.join(msgs))
+    try:
+        do_command(cmd=cmd, show_console=verbose)
+    except RuntimeError as e:
+        raise CmdStanInstallError(
+            'Command "make clean-all" failed\n%s' % str(e)
```

Comment:
While we did disable the pylint warning, I do think we should try to use f-strings going forward for this type of thing
```suggestion
           f'Command "make clean-all" failed\n{e}'
```